src/components/TreeImageScrubber.js

```diff
@@ -767,59 +710,135 @@ function SidePanel(props) {
-                value="palm"
-                control={<Radio />}
-                label="Palm"
-              />
-              <FormControlLabel
-                onClick={() => setCaptureApprovalTag('timber')}
-                value="timber"
-                control={<Radio />}
-                label="Timber"
-              />
-            </RadioGroup>
+            <>
+              <Grid>
+                <Typography className={classes.sidePanelItem} variant="h6">
+                  Morphology
+                </Typography>
+              </Grid>
+              <Grid
+                className={`${classes.bottomLine} ${classes.sidePanelItem}`}
+              >
+                <RadioGroup value={morphology} className={classes.radioGroup}>
+                  <FormControlLabel
+                    value="seedling"
+                    onClick={() => setMorphology('seedling')}
+                    control={<Radio />}
+                    label="Seedling"
+                  />
+                  <FormControlLabel
+                    value="direct_seedling"
+                    control={<Radio />}
+                    onClick={() => setMorphology('direct_seedling')}
+                    label="Direct seeding"
+                  />
+                  <FormControlLabel
+                    onClick={() => setMorphology('fmnr')}
+                    value="fmnr"
+                    control={<Radio />}
+                    label="Pruned/tied (FMNR)"
+                  />
+                </RadioGroup>
+              </Grid>
+              <Grid
+                className={`${classes.bottomLine} ${classes.sidePanelItem}`}
+              >
+                <Typography variant="h6">Age</Typography>
+                <RadioGroup value={age} className={classes.radioGroup}>
+                  <FormControlLabel
+                    onClick={() => setAge('new_tree')}
+                    value="new_tree"
+                    control={<Radio />}
+                    label="New tree(s)"
+                  />
+                  <FormControlLabel
+                    onClick={() => setAge('over_two_years')}
+                    value="over_two_years"
+                    control={<Radio />}
+                    label="> 2 years old"
+                  />
+                </RadioGroup>
+              </Grid>
+              {/*
+        <Grid className={`${classes.bottomLine} ${classes.sidePanelItem}`}>
+          <RadioGroup className={classes.radioGroup}>
+            <FormControlLabel disabled value='Create token' control={<Radio/>} label='Create token' />
+            <FormControlLabel disabled value='No token' control={<Radio/>} label='No token' />
+          </RadioGroup>
+        </Grid>
+        */}
+              <Grid>
+                <Typography className={classes.sidePanelItem} variant="h6">
+                  Species (if known)
+                </Typography>
+                <Species />
+              </Grid>
+              <Grid>
```

Comment:
The _Additional tags_ should be available for both _Approve_ and _Reject_ operations.

Reply:
Makes sense. I added them to both now!